packages/cozy-harvest-lib/src/connections/triggers.js

```diff
@@ -50,8 +50,7 @@ const waitForLoginSuccess = async (
   const jobSubscription = await subscribe(
     {
       token: client.client.token.token,
```

Comment:
something to change in the future, it seems very weird :s

Reply:
See it like a song.